deep_trekker.orogen

```diff
@@ -21,9 +21,8 @@ task_context "RevolutionTask" do
     property "input_timeout", "base/Time"
     # Camera head tilt limits
     property "camera_head_limits", "deep_trekker/CameraHeadLimits"
-    # Minimum speed command for vertical thrusters to counteract buoyancy
-    property "vertical_thrusters_minimum_command", "double"
-
+    # Constant offset command for vertical thrusters to counteract buoyancy
+    property "vertical_thrusters_offset_command", "double"
```

Comment:
Clarify that the offset is always absolute

Reply:
It should not be.

It really should only be "an offset". If the ROV has a negative buoyancy, we should be able to counter it (not that we want to in principle, but ...)

Reply:
Yeah, I ended up figuring it out by looking at the library PR. I had misread the updated code.